watson/watson.py

```diff
@@ -443,19 +443,27 @@ def merge_report(self, frames_with_conflict):
 
         return conflicting, merging
 
+    def _validate_report_options(self, filtrate, ignored):
+        return bool(
+            filtrate and ignored and set(filtrate).intersection(set(ignored)))
```

Comment:
I think that if a function "validate" something, it should return True when it's valid, not the opposite :) .

Reply:
fair enough :P I'll take a look at it tomorrow and push a fixed version :)